### DISCUSSION TITLE
kangaroo/best-practice/insecure_registry

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#4661] [Announcements]
# kangaroo/best-practice/insecure_registry

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/kangaroo/best-practice/insecure_registry.html

**Comment by san3Xian** [comment]
[plugins."io.containerd.grpc.v1.cri".registry]
  config_path = "/etc/containerd/certs.d"
默认是没有配置的(可用命令``crictl info |grep configPath -i -C 10``确认)
<img width="528" alt="image" src="https://github.com/DaoCloud/DaoCloud-docs/assets/24935499/7c0ab36d-521a-415a-8401-3b83b4131e65">


而且，如果定义了config_path，同一个文件下就不能定义mirrors，否则containerd会无法启动
<img width="1680" alt="image" src="https://github.com/DaoCloud/DaoCloud-docs/assets/24935499/d007b858-b7a1-403e-8a20-40372aef5b72">
